samlsp/middleware.go

```diff
@@ -52,13 +53,14 @@ type Middleware struct {
 	AllowIDPInitiated bool
 	CookieName        string
 	CookieMaxAge      time.Duration
+	JwtSigningMethod  jwt.SigningMethod
+	JwtSigningKey     crypto.PrivateKey
+	JwtCheckKey       crypto.PublicKey
```

Comment:
nit: renaming Check -> Verify might be more clear.

Reply:
I will update PR, thanks for the feedback